src/chrome/content/rules/INYT.com.xml

```diff
@@ -1,57 +1,44 @@
 <!--
 	For other New York Times coverage, see NYTimes.xml.
 
+	Incomplete cert chain:
+		customercare.inyt.com	http://customercare.inyt.com/ManageMyAccount/Login.html
+		partner.inyt.com	http://partner.inyt.com/ExtranetPnD/login
+		subscribe.inyt.com	http://subscribe.inyt.com/student
 
-	(www.)?inyt.com: Mismatched
-
+	Mismatched:
+		(www.)?inyt.com
+		eedition.inyt.com
 
 	Insecure cookies are set for these hosts:
-
 		- customercare.inyt.com
 		- subscribe.inyt.com
-
 -->
 <ruleset name="INYT.com (partial)">
+	<!--	Direct rewrites:	-->
+	<target host="goldengate.inyt.com" />
+		<test url="http://goldengate.inyt.com/dana-na/auth/remediate.cgi" />
 
-	<!--	Direct rewrites:
-				-->
-	<target host="customercare.inyt.com" />
-	<target host="subscribe.inyt.com" />
-
-	<!--	Complications:
-				-->
+	<!--	Complications:	-->
 	<target host="inyt.com" />
 	<target host="www.inyt.com" />
-
-		<!--	/*(?!$|\?) 404s:
-					-->
+		<!--	/*(?!$|\?) 404s:	-->
 		<exclusion pattern="^http://(?:www\.)?inyt\.com/+(?!$|\?)" />
-
-			<!--	+ve:
-					-->
+			<!--	+ve:	-->
 			<test url="http://www.inyt.com/digileftnav" />
 			<test url="http://www.inyt.com/edu-leftnav" />
 			<test url="http://www.inyt.com/share/" />
 			<test url="http://www.inyt.com/subscriptions/inyt/lp3KXL9.html" />
 			<test url="http://www.inyt.com/subscriptions/inyt/lp3KYXX.html/" />
 			<test url="http://www.inyt.com/subscriptions/inyt/lp3L3W7.html" />
 
-
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^(?:customercare|subscribe)\.inyt\.com$" name="^JSESSIONID$" /-->
-
 	<securecookie host="^\." name="^(?:__utm|_ga$)" />
 	<securecookie host="^\w" name="." />
 
-
 	<!--	Redirect keeps args but
-		not forward slash:
-					-->
+		not forward slash:	-->
 	<rule from="^http://(?:www\.)?inyt\.com/+"
```

Comment:
Maybe we need to get rid of this redirect completely. It seems only to work inside `http://www.inyt.com/subscriptions/`, but then you have mixed content issues on `global.nytimes.com`.